src/avram/database.cr

```diff
@@ -156,7 +156,7 @@ abstract class Avram::Database
   end
 
   # :nodoc:
-  def transaction : Bool
+  def transaction
```

Comment:
Not too sure about this change. In theory, if it's ok, then we should be able to remove [this line](https://github.com/luckyframework/avram/blob/master/src/avram/save_operation.cr#L299) and still have things work. I pulled down this PR, and got 3 failures on SaveOperations.